CI/unit_tests/utils/test_testing.py

```diff
@@ -0,0 +1,109 @@
+"""
+MDSuite: A Zincwarecode package.
+
+License
+-------
+This program and the accompanying materials are made available under the terms
+of the Eclipse Public License v2.0 which accompanies this distribution, and is
+available at https://www.eclipse.org/legal/epl-v20.html
+
+SPDX-License-Identifier: EPL-2.0
+
+Copyright Contributors to the Zincwarecode Project.
+
+Contact Information
+-------------------
+email: zincwarecode@gmail.com
+github: https://github.com/zincware
+web: https://zincwarecode.com/
+
+Citation
+--------
+If you use this module please cite us with:
+
+Summary
+-------
+Test the mdsuite testing modules.
+"""
+import time
+import unittest
+
+import numpy as np
+
+from mdsuite.utils.testing import MDSuiteProcess, assertDeepAlmostEqual
+
+
+class TestProcess(unittest.TestCase):
+    """
+    Test the process class
+    """
+
+    def test_exception(self):
+        """
+        Test a raised exception.
+
+        Returns
+        -------
+
+        """
+        process = MDSuiteProcess(target=self._exception_throw)
+        process.start()
+        time.sleep(5)
+        process.terminate()
+        self.assertEqual(True, process.exception is not None)
+
```

Comment:
`assert True` should be sufficient instead of `assert True == True`.

Reply:
resolved in d8c6389